### PR TITLE
feat: update interface to use init control command

### PR DESCRIPTION
**Pull request by idobes**

### CODE REVIEW

File: src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/worker.py [hunk 1]
```diff
@@ -19,7 +20,7 @@ class Worker(ImageDetector, ABC):
     method.
     """
 
-    def __init__(self, image_queue: Deque, sio, **kw):
+    def __init__(self, image_queue, sio, **kw):
```
Comment (Kapim) [inline src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/worker.py:23]: put there annotation for image_queue and sio please

File: src/ros1/era_5g_object_detection/src/node.py [hunk 2]
```diff
@@ -2,25 +2,26 @@
...
+
+task_handler = TaskHandlerInternalQ("ros_node", image_queue, if_queue_full="discard_oldest")
 
 INPUT_TOPIC = os.getenv("INPUT_TOPIC", None)
 OUTPUT_TOPIC = os.getenv("OUTPUT_TOPIC", None)
 
-
 class ObjectDetector(MMDetectorWorker):
-    def __init__(self, image_queue: Deque, pub, **kw):
+    def __init__(self, image_queue, pub, **kw):
```
Comment (Kapim) [inline src/ros1/era_5g_object_detection/src/node.py:24]: annotations please

**Review by Kapim — CHANGES_REQUESTED** [review]

File: src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py [hunk 3]
```diff
@@ -225,8 +225,32 @@ def json_callback_websocket(sid, data):
 
 @sio.on('command', namespace='/control')
 def command_callback_websocket(sid, data: Dict):
-    command = ControlCommand(**data)
-    if command and command.cmd_type == ControlCmdType.SET_STATE:
+    eio_sid = sio.manager.eio_sid_from_sid(sid, '/control')
+    
+    try:
+        command = ControlCommand(**data)
+    except Exception as e:
```
Comment (ZdenekM) [inline src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py:232]: Please use more specific exception.

Comment (idobes) [reply src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py]: Changed to TypeError.

File: src/ros1/era_5g_object_detection/src/node.py (hunk 2 above)
Comment (idobes) [reply src/ros1/era_5g_object_detection/src/node.py]: Added

File: src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/worker.py (hunk 1 above)
Comment (idobes) [reply src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/worker.py]: Added

**Review by ZdenekM — APPROVED** [review]

**Review by Kapim — APPROVED** [review]

**Pull request merged**